### DISCUSSION TITLE
Scheduling

### DISCUSSION THREAD

**Discussion by thegiantduckling** [shadowmoose/RedditDownloader#206] [Q&A]
Not been able to find this in the wiki or the settings. Is it possible to schedule download once a day for example?

Thanks

**Comment by shadowmoose** [comment]
Hey, thanks for the question. RMD does not currently support this directly. However, RMD can be launched from the command line without needing an interface, so it is recommended that you schedule a CRON job if you would like to automate the downloading process.
